src/main/java/util/General.java

```diff
@@ -0,0 +1,13 @@
+package util;
+
+public class General {
+	public static String strReplace(String s, char searched, char replacement) {
+		char[] chars = s.toCharArray();
+		StringBuffer out = new StringBuffer(chars.length);
+		for (int i = 0; i < chars.length; i++) {
+			char c = chars[i];
+			out.append(c == searched ? replacement : c);
+		}
+		return out.toString();
+	}
+}
```

Comment:
[it's not like String offers a replace-method or something like that xd](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/lang/String.html#replace(char,char))

Reply:
oh you're right, I forgot that xD

Reply:
Using the that interestingMethodForStringManipulation now ;) I'm sure @MalteRichert would like that too xd

Reply:
I love that I don't even have to say anything, my reputation is doing my work for me 🚀